dashboard/modules/log/log_agent.py

```diff
@@ -63,8 +63,15 @@ async def ListLogs(self, request, context):
                 "It is unexpected. Please report an issue to Ray Github."
             )
         log_files = []
-        for p in path.glob(request.glob_filter):
-            log_files.append(p.name)
+        # Traverse into subdirectory.
+        for p in path.rglob(request.glob_filter):
```

Comment:
Btw eventually we'd like to allow users to navigate subdirs. Can you add the work item to the doc?